src/main/resources/config/liquibase/changelog/20210812152000_changelog.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+    <changeSet author="schlesinger" id="20210812152000">
+        <addColumn tableName="post">
+            <column name="is_pinned" type="boolean" defaultValueBoolean="false"/>
+            <column name="is_archived" type="boolean" defaultValueBoolean="false"/>
```

Comment:
If we can still adjust it, it might be a good idea to make these not nullable, as nullable booleans always tend to produce issues with null. If the change is already on the servers, maybe a second change log entry. Just came to my mind when seeing this.

Reply:
what do you mean with "on the servers"? I could still adjust is in this PR.

Reply:
the PR has been deployed on test servers...

Reply:
Otherwise, we should compare `Boolean` always with `Boolean.TRUE.equals(value)` to avoid NullPointerExceptions due to auto-unboxing.

Reply:
Good comment! I can update the test server manually no problem.

Reply:
Sorry, haven't seen all your comments while commenting. Yes, I mean the test servers, and primitive boolean would be preferred, at least in the DB. An adjustment would be great.

Reply:
Thanks, I'll prepare a commit with the suggested changes